pkg/controller/mcmhub/ansiblejob.go

```diff
@@ -70,13 +73,59 @@ func (jIns *JobInstances) registryJobs(subIns *subv1.Subscription, suffixFunc Su
 
 		nx := ins.DeepCopy()
 		suffix := suffixFunc(subIns)
+
+		if forceRegister {
```

Comment:
I think the naming should be something like this, which can make the `force` job unique

```
		suffix := suffixFunc(subIns)

		if forceRegister {
			plrSuffixFunc := func(a *subv1.Subscription) string {
				return fmt.Sprintf("-%v-%v", subIns.GetGeneration(), placementRuleRv)
			}

			suffix = plrSuffixFunc(subIns)
		}

		nx.SetName(fmt.Sprintf("%s%s", nx.GetName(), suffix))
		}
```